### DISCUSSION TITLE
How the boiler interprets parameter 730, 1030 for heating circuits?

### DISCUSSION THREAD

**Discussion by GSzabados** [fredlcore/BSB-LAN#401] [Q&A] [answered]
Hi,

So it is time to switch to winter mode as the temperature dropped to close to 0 outside during the night. I've had both 730 and 1030 set to 11 Celsius, but I cannot see that the boiler would really kick in when the temperature drops bellow 11 outside.
How are these settings supposed to work with the boiler if any one of you know.
I guess it is using the "attenuated"  (8703) outside temperature, but really I have no idea how is that calculated and what limit the temperature has to pass to swicth from summer to winter or winter to summer.
As it is more hardware (boiler) related, I hope @1coderookie can share some of the knowledge what he has about these units.

Thanks in advance! 

Gábor Szabados

**Comment by 1coderookie** [comment, marked as answer]
Hi,
if I can help, I definitely will :) 
Whats the type, model an controller you are using btw? Did you send us the output of /Q? Cuz I don't find you in my userlist..

Well, in general the parameters were right, as they define the value for the summer/winter mode. And yes, I think it's 8703 that counts. So it _can_ happen in this period, that it's too warm during the day (before) and that the boiler doesn't kick in because of that. But if the temps drop down to 0°, this shouldn't occur of course..
So first, try to set this value (730&1030) HIGHER, e.g. to 16°. That means, that the boiler will switch from summer to winter if the 8703-temp drops below 16°. Right now it has to drop below 11° with your setting, so that may be a little too low. 
On the other hand, the heater will switch to summermode later - if the 8703-temp is above 16° (in this example) - so it will run/be willing to run until that outside temps will be reached again.

Btw: whats the temp value you set in 714&1014? 

Cheers

**Reply by 1coderookie** [reply]
For clarification:
The lower you set the 730/1030 value, the later the heater switches on to wintermode from summer and the earlier it switches from summer to winter. But: That only means, that the mode "automatic" is active or inactive. That means, that when it's active (winter mode), the heater follows your settings from the time schedule. If its inactive (summer mode), then the heater won't fire up (besides DHW, if you enabled DHW mode). 
You can check which mode (summer/winter) is active though if you look at the display of the operational unit (ISR) at the heater, but for this the mode "automatic" must be chosen: If there is the black line (right above the timeline at the bottom of the display) visible which indicates your timeschedule, then the wintermode is active. If there's only the fine timeline but no bigger black line above, then the summermode is active. 
You can check out the picture of the [QAA75 here](https://1coderookie.github.io/BSB-LPB-LAN/kap10.html#1052-qaa75--qaa78) for example, because maybe my description isn't really clear: There you can see the thick black line for the time schedule, which is about from 6am to 12:30pm and from 2pm-10pm in that case. 
I hope you get what I mean.. ;)
So in that shown case, the wintermode is active with the two mentioned time frames where it should heat up to the comfort temperature.

**Reply by GSzabados** [reply]
> I hope you get what I mean.. ;)

Yes, I've got all what you mean. Then I might have asked my question incorrectly. 

Automatic set, as you described. 730 and 1030 was set to 11, then I increased to 12 when 8703 was somewhere around 11.4, but the heater hasn't kicked in as I would expect.

How is 8703 calculated and when triggers the heater? I had a similar experience during spring when it was commissioned. I had to change 730 and 1030 to 11 C, to force turn off the heater, when the house was already too hot. 

Here to add, I don't have a room unit but I might going to change this soon with a temp sensor and feed it into BSB-LAN. I don't really get how the heating setpoint works without a measurement at the place where the heat is produced for (ie.: living room).

**Reply by 1coderookie** [reply]
Well, how _8703 is calculated is described in the manual of the heater iirc. That's somewhat like the average value of the last day(s?) - I don't know exactly though, I read it and then I forgot ;P
It also doesn't really trigger the heater itself (imho), it's just more like the temp value which e.g. counts for switching between summer/winter. So if summer is active because your 730 value is too low and therefore the heater gets the info "too warm outside, must be summer, so I just don't fire up", then it just never kicks in. If 730 is set higher though (e.g. 16°) and 8703 is always underneath it, then the heater knows "uuh, too cold for summer, must get ready to fire up" and switches to winter. Then it depends of your time schedule in automatic mode for example if and when the heater actually starts firing. 
So basically you only need 730 to determine once when this summer/winter switch should occur. During winter runtime, you could just use the presence button to temporarily switch to reduced mode in automatic during the day when your house is warm enough - or just change the whole mode by itself to reduced or so (button at the top right corner), if you want to have that persistent.

The 'heating setpoint' how you call it (if I get you right here) is looking at e.g. the boiler temperature of the heater (_again: which kind of heating system and which controller do you have?_). E.g. you are having an oil burner: If the time schedule says "(comfort) heating time", then the HC-pump is running an the boiler temp will drop sooner or later. Then the burner kicks in and fires up to a certain temperature, which is dependent on the settings of the heating curve and so on. The colder it is outside, the higher the desired boiler temp will be basically. (A gas fired burner works more or less the same, just with the difference that it can modulate, that means it can als run 24/7 with just a very little flame, whereas the oil burner just fires up until a certain boiler temp is reached and then it shuts off again.)
So the system doesn't actually 'know' which temp you have inside your room(s) because of the missing room temp sensor, it's more like 'guessing' (based upon certain parameters) which temps are desired..
Grab your manual for the heating system and try to go through it, it should all be described there.

**Reply by GSzabados** [reply]
> Grab your manual for the heating system and try to go through it, it should all be described there.

Ok, I will do, but I need a bit of help with that as well. I do have a Polish manual, but I don't speak Polish. :laughing: 

Can you give me a hand with this? I mean, a link to an English manual. I tried before the Brötje website, but I wasn't successful as I remember, back in this Spring. 

You asked me before about the parameters, and the heater. Yes, I did send the parameter query to @fredlcore by email once I installed the BSB-LAN module. And I think, I wrote before in another discussion, that I have a Brötje WBS 22 i with an ISR Plus (ISR-LMS). It is a gas system and has 2 HC, 1 is with radiators, 2 is floor heating, plus HC4 is DHW with a tank. (I think we last discussed about the heater back in August, when I was fiddling to add the DHW to Home Assistant as a switch.)

Thanks for all the descriptions. It all make sense, and it is how I was thinking, that how it should work. But I find it a bit weird, that compared to the old system, it hasn't got an option to manually set Summer or Winter. The Comfort and Reduced option on the LMS is not equivalent to that. As I understand the heater will turn on only, even if it is in Comfort or Reduced, if the temperature is below the Summer/Winter change temperature. Please correct me if my logic is wrong. The old heater had a physical switch for Summer/Winter change, but not the automatic. And Comfort and Reduced were set from the controller, which was a Viessman controller.

**Reply by 1coderookie** [reply]
Ah ok, so maybe he sent me that query for the user listing and I'm just not aware of your name you used there and your name here.. 
And yes, you probably answered my question about your heating system back then already, but I/we have so many questions etc coming in, that I just don't remember every single user and his system. ;)
So, the best would be if you'd send me an email, so that we can discuss this further in private. Would be good if you would then also send me the output of Q and probably also the output of a complete query (/0-10000), each saved and attached as a txt-file. That way I'd have all the informations I'd need and then I guess I can help you.. ;) 

And to your last question: 
No, (afaik) there isn't a way to manually switch between summer/winter as it is done automatically. 
You could switch between the mentioned modes (automatic/comfort/reduced/protection), but that's not the same like summer/winter as you already correctly mentioned. Within the automatic mode you could use the presence button for the temporary mode switch between comfort/reduced (which will be active until the next mode change will occur due to the time schedule), but this would only be possible if the heater is in winter mode and therefore the automatic mode is active already (as mentioned before: you can check this by looking at the display and check if the big black line which indicates the scheduled times is apparent or not). 

But in one thing you're wrong: summer/winter _doesn't_ affect comfort/reduced! _Even if the heater is in summer mode, you can still turn it on by switching to the main mode "comfort" (by using button at the top right corner of the ISR operation unit or by using BSB-LAN of course)! This summer/winter switch only affects the automatic mode!_ 

Just as an example: I personally set the value to 16° iirc, but also with that it sometimes happens that it's cold at one day and the heater isn't ready and the next day it's warm and the heater is firing up due to the low value of the day(s) before. So that actually kinda sucks in these periods where we go from summer to winter, but then I just switch the mode itself by using the main mode button and set it to comfort instead of automatic that day to turn it on. Same goes with the period winter to summer: If it's a day where it's warm outside already and I don't need the heater but it's firing itself up due to the cold temperature(s) the day(s) before, then I just switch it 'off' by changing the mode to "reduced" instead of automatic (but I have to mention that I set the parameters for reduced in a way that the heater doesn't fire up, so that kinda depends on the individual settings if you have to use 'reduced' or 'protection' mode!)

**Reply by GSzabados** [reply]
Thank again for the reply and the help. I've just found your email address as well, so I will send you an email during the week with all the details.

**Reply by 1coderookie** [reply]
I just also found you and sent you an email already.. ;)
